contracts/Strategy.sol

```diff
@@ -489,11 +497,12 @@ contract TrancheStrategy is BaseStrategy {
 
         // work successfully
         if (checkStakedBeforeMigrating) {
-            protected = new address[](length + 1);
+            protected = new address[](length + 2);
             protected[length] = address(tranche);
+            protected[length + 1] = address(gauge);
         } else {
             // escape hatche as we can during an emergency.
-            // skip protecting tranche
+            // skip protecting tranche and gauge token
             protected = new address[](length);
         }
```

Comment:
is this necessary? Governance is the only one that can call `sweep`, which is the only function that calls `protectedTokens`, so it would seem like this part of the code will never be reached practically

Reply:
Actually it's unnecessary. I could't agree more.
I will remove that part oft he code.